force-app/main/default/classes/SOQL.cls

```diff
@@ -183,7 +241,8 @@ public with sharing class SOQL {
      * @return A list of Opportunities with an Amount greater than 100,000, ordered by CloseDate, and limited to 5 records.
      */
     public static List<Opportunity> getHighValueOpportunities() {
-        return null; // Replace null with the variable you used to store the result
+        List<Opportunity> opps = [SELECT Name, AccountId, CloseDate, StageName, Amount FROM Opportunity WHERE Amount > 100000 ORDER BY CloseDate LIMIT 5]; 
+        return opps; // Replace null with the variable you used to store the result
```

Comment:
This is where you'll want to start thinking about query formatting. As your queries get bigger and more complex, you will want to start formatting them vertically so they are easier to read. Here's an example from an earlier question:
```suggestion
        List<Opportunity> opps = [
                SELECT 
                        Id,
                        Name,
                        StageName,
                        AccountId,
                        CloseDate,
                        Amount 
                FROM Opportunity 
                WHERE AccountId = :accId 
                AND StageName = 'Closed Won'
        ];
```